notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>In a 2D spectrum, x and y are used for plotting. That's not relevant for a 1D spectrum, but it is confusing to see and thus worth clarifying.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>